wasmsdk/wallet_base.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+	"github.com/0chain/errors"
+	"github.com/0chain/gosdk/core/zcncrypto"
+)
+
+var _config localConfig
```

Comment:
The `_config` is defined in zcncore package, what we need to do is just call the `SplitKeys` and `SetWalletInfo`.